From fb9a90622fec7d8207962b7e8b58dfe8decc94ae Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Fri, 11 Jul 2014 14:20:59 +0200
Subject: [PATCH 25/43] xhci: switch debug printf to tracepoint

Message-id: <1405088470-24115-27-git-send-email-kraxel@redhat.com>
Patchwork-id: 59853
O-Subject: [RHEL-7.1 qemu-kvm PATCH 26/37] xhci: switch debug printf to tracepoint
Bugzilla: 980747
RH-Acked-by: Dr. David Alan Gilbert (git) <dgilbert@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
(cherry picked from commit 4f9cc7342281bfbbc1bfccc155c59551c06d63db)
---
 hw/usb/hcd-xhci.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/usb/hcd-xhci.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/hw/usb/hcd-xhci.c b/hw/usb/hcd-xhci.c
index 75da33d..f937b4e 100644
--- a/hw/usb/hcd-xhci.c
+++ b/hw/usb/hcd-xhci.c
@@ -2041,9 +2041,7 @@ static int xhci_submit(XHCIState *xhci, XHCITransfer *xfer, XHCIEPContext *epctx
         }
         break;
     default:
-        fprintf(stderr, "xhci: unknown or unhandled EP "
-                "(type %d, in %d, ep %02x)\n",
-                epctx->type, xfer->in_xfer, xfer->epid);
+        trace_usb_xhci_unimplemented("endpoint type", epctx->type);
         return -1;
     }
 
-- 
1.8.3.1

